src/client/styles/Grid.ts

```diff
@@ -93,24 +93,27 @@ export enum MAX_WIDTH {
   WIDE = maxWidth(COLUMNS.WIDE, 60, space[5], space[5]),
 }
 
-const generateGridPaddingCss = (padding: number, maxWidth?: number) => `
-  padding-left: ${px(padding)};
-  padding-right: ${px(padding)};
-  max-width: ${maxWidth ? px(maxWidth) : '100%'};
-`;
-
 const generateGridRowCss = (
   padding: number,
   columnWidth: string,
   columnNumber: number,
+  maxWidth?: number,
 ) => `
     -ms-grid-columns: (${columnWidth} 20px\)[${
   columnNumber - 1
 }] ${columnWidth};
     grid-template-columns: repeat(${columnNumber}, ${columnWidth});
+    padding-left: ${px(padding)};
+    padding-right: ${px(padding)};
+    max-width: ${maxWidth ? px(maxWidth) : '100%'};
 `;
 
-export const innerGridRow = css`
+// this styles should be applied to an element to make it behave as
+// the grid container, it defines the css on how all the breakpoints
+// should behave
+// anything items should be added inside this container, and use
```

Comment:
overall reads well, just wondering about maybe clearing up this part: "anything items.."

Reply:
I should mention that src/client/styles/Grid.ts is actually completely unchanged now that I'm using the correct design.